Database/compiler.lua

```diff
@@ -540,13 +545,22 @@ QuestieDBCompiler.writers = {
             QuestieDBCompiler.writers["objective"](stream, value[2])
             QuestieDBCompiler.writers["objective"](stream, value[3])
             QuestieDBCompiler.writers["u24pair"](stream, value[4])
-            if value[5] and value[5][1] and value[5][1][1] then
-                stream:WriteByte(#value[5][1])
-                for _, v in pairs(value[5][1]) do
-                    stream:WriteInt24(v)
+
+            local killobjectives = value[5]
+            if type(killobjectives) == "table" and #killobjectives > 0 then
```

Comment:
Same as before, right?

```suggestion
            if killobjective and #killobjectives > 0 then
```

Reply:
Not so sure about this one, when I tried to code this on my simpler checks didn't seem to work here, but maybe I did something wrong. @Laumesis might be able to say more on this?